packages/react-log-viewer/src/LogViewer/LogViewer.tsx

```diff
@@ -188,12 +188,12 @@ const LogViewerBase: React.FunctionComponent<LogViewerProps> = memo(
       setListKey(listKey => listKey + 1);
     };
 
-    const scrollToRowInFocus = (searchedRowIndex: number) => {
+    const scrollToRowInFocus = (searchedRowIndex: searchedKeyWordType) => {
       setRowInFocus(searchedRowIndex);
-      logViewerRef.current.scrollToItem(searchedRowIndex, 'center');
+      logViewerRef.current.scrollToItem(searchedRowIndex.rowIndex, 'center');
       // use this method to scroll to the right
       // if the keyword is out of the window when wrapping text
-      if (!isWrapText) {
+      if (!isTextWrapped) {
         setTimeout(() => {
           const element = document.querySelector('.pf-c-log-viewer__string.pf-m-current');
```

Comment:
do you have a ref to the logviewer parent?
would be better to use querySelector on the parent ref. I am thinking of the case where there are 2 log viewers on the same page, if you search on the second one it will cause the window to jump up to the first one otherwise